### PR TITLE
Fix Issue #5

### PR DESCRIPTION
**Pull request by brownie1**
Added the chown command for the osd0 and osd1 directories.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,11 +103,11 @@ vagrant@ceph-admin:~/test-cluster$ ceph-deploy mon create-initial
 And our two OSDs. For these, we need to log into the server machines directly:
 
 ```console
-vagrant@ceph-admin:~/test-cluster$ ssh ceph-server-2 sudo mkdir /var/local/osd0
+vagrant@ceph-admin:~/test-cluster$ ssh ceph-server-2 sudo "sudo mkdir /var/local/osd0 && sudo chown ceph:ceph /var/local/osd0"
```
Comment (carmstrong) [inline README.md:106]: Looks like there is an extra `sudo` here.

Comment (brownie1) [inline README.md:106]: Running a test local when I run the following command:
sudo mkdir /tmp/abc && chown brownie:brownie /tmp/abc

I get a permission denied error. If I running the command with a sudo on the front of the chown then it works. The && is like running two separate commands so you need the extra sudo.

Comment (carmstrong) [inline README.md:106]: Cool, thanks. Can you rebase and I'll merge?

**Comment by carmstrong** [comment]
@brownie1 It looks like this was resolved in #7 - could you give master a try and see if you fare better? Thanks!

**Pull request closed without merging**